gravitee-apim-definition/gravitee-apim-definition-jackson/src/main/java/io/gravitee/definition/jackson/datatype/api/deser/ApiDeserializer.java

```diff
@@ -78,12 +80,14 @@ public T deserialize(JsonParser jp, DeserializationContext ctxt, T api, JsonNode
         }
         api.setExecutionMode(ExecutionMode.fromLabel(node.path("execution_mode").asText()));
 
-        JsonNode proxyNode = node.get("proxy");
-        if (proxyNode != null) {
-            api.setProxy(proxyNode.traverse(jp.getCodec()).readValueAs(Proxy.class));
-        } else {
-            logger.error("A proxy property is required for {}", api.getName());
-            throw JsonMappingException.from(ctxt, "A proxy property is required for " + api.getName());
+        if (api.getDefinitionVersion() != DefinitionVersion.FEDERATED) {
+            JsonNode proxyNode = node.get("proxy");
+            if (proxyNode != null) {
+                api.setProxy(proxyNode.traverse(jp.getCodec()).readValueAs(Proxy.class));
+            } else {
+                logger.error("A proxy property is required for {}", api.getName());
+                throw JsonMappingException.from(ctxt, "A proxy property is required for " + api.getName());
+            }
```

Comment:
I don't really understand why we have to do this because we should have the same "issue" with V4 as there is no proxy in v4 definition 🤔

Reply:
The error happend after associate API to group but before trigger notification. Maybe we have the problem for V4 without proxy. I will try this.

Reply:
In fact I don't know how to create APIv4 without proxy...

Reply:
Proxy is only relevant for v2 API. V4 don't have proxy attribute

Reply:
I reproduce for V4. I replace the check for enforce only for V2